helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -1705,4 +1734,27 @@ private static void verifyToken(final String token, final Job job)
     }
   }
 
+  private List<TaskStatus.State> getPreviousJobStates(final JobId jobId, final String host) {
+    List<TaskStatus.State> previousStates;
+    try {
+      final List<TaskStatusEvent> jobHistory = getJobHistory(jobId, host);
+      jobHistory.subList(0, Math.min(10, jobHistory.size()));
+      Function<TaskStatusEvent, TaskStatus.State> statusesToStrings =
+          new Function<TaskStatusEvent, TaskStatus.State>() {
+            @javax.annotation.Nullable
```

Comment:
Eh? Should this be here? Also why is it `@javax.annotation.Nullable` instead of just `@Nullable`?